src/components/users/Accounts.jsx

```diff
@@ -45,12 +45,18 @@ export function getAccountsState (
     };
 }
 
+export const cryptUserPassword = async (password) => {
+    const pythonScript = `from random import SystemRandom as sr; import crypt; print(crypt.crypt("${password}", "$y$j9T$" + "".join(sr().choice("./0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz") for _sc in range(24))))`;
```

Comment:
Please move this script in to scripts/myscript.py and importit.

You can then run it from JS with:
   import myscript from '../scripts/myscript.py';                                       

 python.spawn(myscript, args, { err: "message", environ: ['LC_ALL=C.UTF-8'] });